metaflow/plugins/cards/card_decorator.py

```diff
@@ -117,6 +117,7 @@ def step_init(
         # `'%s-%s'%(evt_name,step_name)` ensures that we capture this once per @card per @step.
         # Since there can be many steps checking if event is registered for `evt_name` will only make it check it once for all steps.
         # Hence, we have `_is_event_registered('%s-%s'%(evt_name,step_name))`
+        self._is_runtime_card = False
```

Comment:
Driveby change to ensure that if something crashes at the task-pre-step level then then the card decorator doesn't add an additional exceptions to the error trace. Since `self._is_runtime_card` used to be set in task-pre-step and if something fails before that executes for the card decorator then the card decorator's task finished threw one more exception saying that `_is_runtime_card`  is not present.